tasks/cluster_agent.py

```diff
@@ -210,11 +217,11 @@ def hacky_dev_image_build(ctx, base_image=None, target_image="cluster-agent", pu
 '''
         )
         dockerfile.flush()
-
+        target_image_name = f'{target_image}:{target_tag}'
         pull_env = {}
         if signed_pull:
             pull_env['DOCKER_CONTENT_TRUST'] = '1'
-        ctx.run(f'docker build -t {target_image} -f {dockerfile.name} .', env=pull_env)
+        ctx.run(f'docker build -t {target_image_name} -f {dockerfile.name} .', env=pull_env)
 
         if push:
             ctx.run(f'docker push {target_image}')
```

Comment:
```suggestion
            ctx.run(f'docker push {target_image_name}')
```
I suppose we want to push with the tagged name, correct?

Reply:
Thanks for catching this